### PR TITLE
File structure cleanup

### PR DESCRIPTION
**Pull request by gregmagolan**
Cleanup of /internal folder. Put rules into appropriate folders.

/cc @alexeagle

### CODE REVIEW

**Comment by gregmagolan** [comment]
Ran clang-format and buildifier. clang-format is now a precommit check and buildifier runs in CI. Also fixed all skylint complaints but "preskylint" crashes the JVM in CI so I didn't add it to CI yet.

**Comment by gregmagolan** [comment]
Skylint now running in CI and all the complaints are cleaned up. Running skylint out of the docker container instead of building it in CI.

File: internal/common/expand_into_runfiles.bzl [hunk 1]
```diff
@@ -0,0 +1,19 @@
+def expand_location_into_runfiles(ctx, path):
+  """If the path has a location expansion, expand it. Otherwise return as-is.
+  """
+  if path.find('$(location') < 0:
+    return path
+  return expand_path_into_runfiles(ctx, path)
+
+def expand_path_into_runfiles(ctx, path):
```
[the remark this answers is not in the document]
Comment (alexeagle) [reply internal/common/expand_into_runfiles.bzl]: yup that's fine

File: internal/node/node_loader.js [hunk 2]
```diff
@@ -182,30 +191,35 @@ function resolveRunfiles(...pathSegments) {
...
     if (maybe) {
-      if (DEBUG) console.error("node_loader: resolved directory", maybe);
+      if (DEBUG)
+        console.error("node_loader: resolved directory", maybe);
       return maybe;
     }
   }
 
-  if (DEBUG) console.error("node_loader: resolved to default path", defaultPath);
+  if (DEBUG)
```
Comment (alexeagle) [inline internal/node/node_loader.js]: I think the linter should complain that this is no longer on one line and should now have braces.
don't need to fix now...

File: internal/rollup_bundle/rollup_bundle.bzl [hunk 3]
```diff
@@ -123,11 +123,11 @@ def run_uglify(ctx, input, output, debug = False, comments = True):
   )
 
 def _rollup_bundle(ctx):
-  rollup_config = write_rollup_config(ctx)
-  run_rollup(ctx, collect_es6_sources(ctx), rollup_config, ctx.outputs.build_es6)
-  run_tsc(ctx, ctx.outputs.build_es6, ctx.outputs.build_es5)
-  run_uglify(ctx, ctx.outputs.build_es5, ctx.outputs.build_es5_min)
-  run_uglify(ctx, ctx.outputs.build_es5, ctx.outputs.build_es5_min_debug, debug = True)
+  rollup_config = _write_rollup_config(ctx)
+  _run_rollup(ctx, collect_es6_sources(ctx), rollup_config, ctx.outputs.build_es6)
```
Comment (alexeagle) [inline internal/rollup_bundle/rollup_bundle.bzl]: angular calls these functions from `ng_rollup_bundle` - could you check this PR against angular/angular before we merge it?

File: internal/node/node_loader.js (hunk 2 above)
Comment (gregmagolan) [reply internal/node/node_loader.js]: right. I forgot to add a .clang-format file so its on the default format. I'll set it to ```Language:        JavaScript
BasedOnStyle:    Google
ColumnLimit:     100
```
I think that style adds braces

**Pull request merged**